### ISSUE TITLE
How to access file data in ver 3.0

### ISSUE THREAD

**Issue by amazingSurge** [shannonmoeller/gulp-hb#35]
Hi there,

In the v2.6.5, we can use options.data.root.file to access the file object on helper register callaback. 
But in the new versions, it throw a error that file is not undefined. 

```
(function() {
  module.exports.register = function(Handlebars) {
    var path = require('path');

    Handlebars.registerHelper("file", function(options) {
      var file = options.data.root.file;
      
      return file.path;
    });
  };
}).call(this);
```

**Comment by shannonmoeller** [comment]
You need to specify the `file` flag which is now `false` by default.

```js
gulp.src('...')
    .pipe(hb({
        file: true
    }))
    .pipe(gulp.dest('...'))

**Comment by shannonmoeller** [comment]
@amazingSurge You inspired a change. I dropped the `{ file: true }` flag and added a `@file` property. In your example you'll access it like this:

```js
    Handlebars.registerHelper("file", function(options) {
        var file = options.data.file;

        return file.path;
    });
```

In a template you'd access it like this (which does the same thing as your helper):

```handlebars
{{@file.path}}
```

**Comment by shannonmoeller** [comment]
Released as v5.0.0.

https://github.com/shannonmoeller/gulp-hb/tree/v5.0.0#-data-variables

**Comment by amazingSurge** [comment]
Thank you! It works.
